epoch1d/src/constants.F90

```diff
@@ -216,6 +216,9 @@ MODULE constants
   REAL(num), PARAMETER :: log_plasma_screen_const_2 = &
       LOG(SQRT(epsilon0 * kb) / q0 * m0 * c * alpha / 1.4_num / h_bar)
 #endif
+#ifdef PARTICLE_SPIN
+  REAL(num), PARAMETER :: anomalous_magnetic_moment = 0.0011614_num
```

Comment:
Shouldn't this be defined in terms of the fine structure constant, rather than this (apparently?) approximate value?